app/components/Modal/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from "react";
+import Close from "@mui/icons-material/Close";
+import { cx } from "@emotion/css";
+import { Text } from "@klimadao/lib/components";
+
+import * as styles from "./styles";
+
+interface Props {
+  title: React.ReactNode;
+  onToggleModal?: () => void;
+  className?: string;
+}
+
+export const Modal: FC<Props> = (props) => {
+  const showCloseButton = !!props.onToggleModal;
+  console.log(showCloseButton);
+  return (
+    <>
+      <div
+        className={cx(styles.modalBackground, {
+          showCloseButton,
+        })}
+        onClick={props.onToggleModal}
+      />
+      <div className={styles.modalContainer}>
+        <div className={cx(styles.modalContent, props.className)}>
```

Comment:
always a dilemma which `div` to put the `props.className` on.

I've seen components get gross
```
props.titleClassName
props.containerClassName
```

but I really can't think of a better way